pages/projects/projects.css

```diff
@@ -0,0 +1,149 @@
+#title {
```

Comment:
Do we need to restyle this?

Reply:
Isn't it in commons already.

Reply:
The file `common.css` has a definition for `.title` and I have no clue where it's being used. `#title` has a separate definition in the separate stylesheets for the pages.